### DISCUSSION TITLE
Support of Docker or Raspberry Pi

### DISCUSSION THREAD

**Discussion by todrees1967** [nzymedefense/nzyme#367] [Ideas]
Hi, is there a support planned for Docker or Raspberry Pi?
Regards,
Torsten

**Comment by andrewbeard** [comment]
I can't speak toward Docker support, but the Raspberry Pi already has full support:
https://www.nzyme.org/documentation/installation-on-raspberry-pi-os

**Reply by lennartkoopmann** [reply]
That's correct! Nzyme is in fact supposed to be deployed on a Raspberry Pi for most use-cases.
